connectors/src/connectors/intercom/temporal/activities.ts

```diff
@@ -227,20 +231,6 @@ export async function syncCollectionActivity({
     dataSourceName: dataSourceConfig.dataSourceName,
   };
 
-  const helpCenter = await IntercomHelpCenter.findOne({
```

Comment:
This is actually checked on the first activity of the workflow, so not needed here.